### ISSUE TITLE
Feature Request: Addition of Logout Button

### ISSUE THREAD

**Issue by dipuharidasan** [viewvc/viewvc#338]
Hello,

I am submitting this as a feature request as I was unable to find any options to enable the logout button.

Since we are providing access to the source code, it seems essential to include a logout button. If there is an existing option to enable this feature, please guide me.

Thanks,
Dipu H

**Comment by futatuki** [comment]
ViewVC uses http authentication to distinguish user, i.e. ViewVC itself does not manage login session.

**Comment by cmpilato** [comment]
Right, as @futatuki says, ViewVC does not manage its own authentication, but rather this is managed by the browser and web server via standard HTTP-level authentication mechanisms.  There's not really an appropriate way for a CGI script to instruct the browser to somehow forget any credentials supplied during that session.  At best, a "logout" page in ViewVC might be able to instruct the user to close his/her browser (which generally disposes of those credentials), but I'm not convinced it's worth the effort to do so.

It's a reasonable request, but I'm closing it on account of technical feasibility.
